src/Akka.Persistence.Linq2Db.Data.Compatibility.Tests/SqlServer/SqlServerSpecSettings.cs

```diff
@@ -14,15 +14,9 @@ private SqlServerSpecSettings()
         {
         }
         
-        public override string JournalTableName => "EventJournal";
-        
-        public override string JournalMetadataTableName => "Metadata";
-        
-        public override string SnapshotTableName => "SnapshotStore";
-        
         public override string ProviderName => LinqToDB.ProviderName.SqlServer2017;
         
-        public override string CompatibilityMode => "sqlserver";
+        public override string TableMapping => "sql-server";
```

Comment:
Since it's relatively low effort in the hocon->class wireup, is it possible to still *allow* the old versions to avoid breaking them? (looking out for some folks who I know use this package that might run into problems if they don't catch it.)

Reply:
Yes, since its just a database name mapping, that is easily doable using hocon substitution